src/style.css

```diff
@@ -0,0 +1,63 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--0 .stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(./images/star-active.svg) no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(n + 2) {
+  height: 16px;
+  width: 16px;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+}
+
+.stars--2 .stars__star:nth-child(n + 3) {
+  height: 16px;
+  width: 16px;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+}
+
+.stars--3 .stars__star:nth-child(n + 4) {
+  height: 16px;
+  width: 16px;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(n + 5) {
+  height: 16px;
+  width: 16px;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+}
```

Comment:
don't repeat styles. set common ones to `stars__star` and change only `background`. Also, you can combine selectors with the same styles